wkconnect/__main__.py

```diff
@@ -28,16 +29,21 @@
 from .webknossos.models import DataSource, DataSourceId, UnusableDataSource
 
 logger = logging.getLogger()
+available_backends: List[Type[Backend]] = [Boss, Neuroglancer, Tiff, Wkw]
+
+
+class ServerContext(SimpleNamespace):
```

Comment:
I picked SimpleNamespace, because that is the default in Sanic: https://github.com/sanic-org/sanic/blob/main/sanic/app.py#L163

Reply:
Ok, might be fine for now. Just tested it, `SimpleNamespace` cannot be type-checked correctly:
```python
from types import SimpleNamespace

class C(SimpleNamespace):
    some_int: int

c = C(some_int=34, a_string="some_string")
print(c.some_int, c.a_string, c.undefined)
```
This yields
```
$ python voxelytics/connect/tools/bla.py
Traceback (most recent call last):
  File "sometest.py", line 7, in <module>
    print(c.some_int, c.a_string, c.undefined)
AttributeError: 'C' object has no attribute 'undefined'
$ mypy voxelytics/connect/tools/bla.py
Success: no issues found in 1 source file
```
A `typing.NamedTuple` (or a normal class) would have the benefit that it can be type-checked statically. I'll add a commit, feel free to revert it.

Reply:
Ok, this doesn't work since Sanic uses the context as well to store more attributes, I'm reverting the changes. Let's keep the `SimpleNamespace`.